en/docs/develop/manage-repository/connect-your-own-github-repository-to-choreo.md

```diff
@@ -44,10 +41,17 @@ Let's connect your GitHub repository and use the REST API implementation in it t
 4. Provide authorization with your GitHub account by clicking  **Authorize with GitHub**. 
 5. If you have not already authorized Choreo apps, click **Authorize Choreo Apps** when prompted.
 6. Select a GitHub account, a repository that includes the implementation, the relevant branch, and the build preset (i.e., **Ballerina** or **Dockerfile**). 
-7. Enter a valid path relative to the root of your repository that points to the implementation of the REST API. If you have not designed and implemented your REST API yet, you can connect an empty repository or a sub-folder and proceed to create the component. 
+7. Enter the following values based on the preset you selected above:
+    - **Ballerina** preset
+        - **Path**: Enter a valid path relative to the root of your repository that points to the implementation of the REST API. If you have not designed and implemented your REST API yet, you can connect an empty repository or a sub-folder and proceed to create the component. 
```

Comment:
Change "empty repository" to "empty GitHub repository".

Reply:
done